recipes/deeptools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: deeptools
+  version: '2.0.0'
+
+source:
+  fn: deepTools-2.0.0.tar.gz
+  url: https://pypi.python.org/packages/source/d/deepTools/deepTools-2.0.0.tar.gz
+  md5: 09821e73f56ddb744a35d01173c6eb29
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```

Comment:
Does it really depend on the numpy ABI? That means deeptools would call numpy C functions instead of the Python interface. If not, you should leave out the x.x.